core/templates/components/state-editor/state-responses-editor/state-responses.component.ts

```diff
@@ -630,8 +630,7 @@ angular.module('oppia').component('stateResponses', {
         $scope.SHOW_TRAINABLE_UNRESOLVED_ANSWERS = (
           SHOW_TRAINABLE_UNRESOLVED_ANSWERS);
         $scope.EditabilityService = EditabilityService;
-        $scope.responseCardIsShown = (
-          !WindowDimensionsService.isWindowNarrow());
+        $scope.responseCardIsShown = true;
```

Comment:
Please add a check for this in the frontend test.

Reply:
I have added the test (for init) at `state-responses.component.spec.ts -> Line 246` ([link](https://github.com/Lawful2002/oppia/blob/2a4b3ebcf56eea3689b4226e2e046141cc818770/core/templates/components/state-editor/state-responses-editor/state-responses.component.spec.ts#L246))